noir-projects/noir-contracts/contracts/app_subscription_contract/src/main.nr

```diff
@@ -39,7 +39,7 @@ pub contract AppSubscription {
 
         note.remaining_txs -= 1;
 
-        storage.subscriptions.at(user_address).replace(&mut note).emit(encode_and_encrypt_note(
+        storage.subscriptions.at(user_address).replace(note).emit(encode_and_encrypt_note(
```

Comment:
Here an elsewhere we're likely now producing warnings for `note` no longer needing to be mutable.